packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -138,6 +144,8 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
 
     let selectedKey = props.defaultSelectedKey !== undefined ? props.defaultSelectedKey : props.selectedKey;
     this._lastReadOnlyAutoCompleteChangeTimeoutId = -1;
+    this._isScrollIdle = true;
+    this._scrollIdleTimeoutId = -1;
```

Comment:
Can we use undefined to denote this not having a value instead of -1? Or is this a pattern elsewhere in Fabric?